me.broken_by.PdfMetadataEditor.yml

```diff
@@ -0,0 +1,35 @@
+app-id: me.broken_by.PdfMetadataEditor
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+commands:
+  - pmedit-cli
```

Comment:
there is no property `commands`. I'm pretty sure flatpak-builder threw a warning when you tried to build this.

Reply:
yes, you are right. 
Thanks, fixed.